main/lib/src/DataCollector.cc

```diff
@@ -195,14 +196,23 @@ namespace eudaq {
             if (m_buffer[i].events.front()->GetEventNumber() == 0)
               EUDAQ_WARN("Event number mismatch > 1 in event " +
                          to_string(ev.GetEventNumber()));
-          }
-        }
-        ev.AddEvent(m_buffer[i].events.front());
-        m_buffer[i].events.pop_front();
-        if (m_buffer[i].events.size() == 0) {
-          m_numwaiting--;
-          more = false;
+         //tests 12/05  
+	  }
         }
+	//tests 12/05 (if)
+	//	if( (m_buffer[i].events.front()->GetEventNumber() == m_eventnumber -1 )  )
+	//	  {
+	    ev.AddEvent(m_buffer[i].events.front());
```

Comment:
Did actually anything in this file change? Commented "test code" should not go into the repo.